core/src/test/java/io/questdb/griffin/UpdateBasicTest.java

```diff
@@ -536,6 +536,68 @@ public void testUpdateNoFilter() throws Exception {
         });
     }
 
+    @Test
+    public void testUpdateDuplicateColumnShouldFail() throws Exception {
+        assertMemoryLeak(() -> {
+            compiler.compile(
+                    "create table tabula as (" +
+                            "   select " +
+                            "     timestamp_sequence(0, 1000000) ts," +
+                            "     cast(x as int) x," +
+                            "     cast(x as int) y" +
+                            "   from long_sequence(5)" +
+                            ") timestamp(ts) partition by DAY",
+                    sqlExecutionContext
+            );
+
+            executeUpdateFails(
+                    "UPDATE tabula SET x = 1, x = 3",
+                    25,
+                    "Duplicate column 'x' in SET clause"
+            );
+
```

Comment:
There are two classes of identifiers : 
- regular - e.g. x, blah, yadayada - that are case insensitive but can't contain whitespace 
- quoted identifiers - "abc", "an id", "look Ma, whitespaces!" - that are case sensitive 

In SQL 'x' is always a text literal, never an identifier, so it would be good to  replace single quotes around identifiers with double quotes in tests .